### DISCUSSION TITLE
Randomly stops working, requiring restart

### DISCUSSION THREAD

**Discussion by jlandd** [pqrs-org/Karabiner-Elements#3835] [Q&A]
I'm using K-E in a very simple way, to bypass a defective shift key on a 2021 MBP M1 Pro, 14.5. The key stays engaged no matter how it is cleaned out and the app has been a Godsend, enabling me to work without sending it off to Apple for service. However, about three times a day it stops functioning, while remaining running, requiring a restart, which puts me back in business until the next time it happens.  Another possibly related issue is that about twice a day, waking from sleep, when logging back on after sleeping the keyboard is totally non responsive, attempting to enter password is as if the keyboard is disconnected. A restart here is needed. 

I can't deduct what might be causing this or how to prevent this behavior.  Any wisdom would be greatly appreciated :)

**Comment by mystagog** [comment]
Hello, my device displays the same behaviour. Still my keyboard does not stop working and the problem is unrelated to sleep. But unlike jlandd I have to do this up to 5 times per hour. 
Any help would be appreciated since it would avoid me paying for a new keyboard that I can hardly afford
